roles/common/vars/main.yml

```diff
@@ -1,3 +1,6 @@
 ---
 # vars file for common
+common_gpg_key_exists: false
+common_convert2rhel_repo_exists: false
+common_convert2rhel_package_exists: false
```

Comment:
Yeah, was talking with Mike Savage the other day, and we basically want to remove the package and the repository at the end of execution, with a success or not, because we want to clean the system. 

Regarding removing the gpg-key, maybe we should leave it in the system as it may be used by some other application (?). But for the package itself and the repo, it's fine to remove, and the collection will download it again after its started (if needed).

Reply:
So I guess we could then get rid of some of the variables and for convert2rhel repo and pkg just remove it everytime, and maybe leave the gpg key on system .. well we can comment out that task and make it 'disabled' and if anyone needs it it can easily be added back

Reply:
I have changed the behavior.  Now it removes them every time. I left the variables so it is easily changeable, but the default is to remove repo and package and leave gpg key in the system.

Reply:
Alright, nice, now all can be changed via variables (will be documented in one of the follow tasks I suppose) and in default gpg key is kept and repo and pkg removed.